Translators/RoadConditions/main.py

```diff
@@ -0,0 +1,138 @@
+import json
+import requests
+import copy
+import logging
+import os
+from pgquery import query_db
+from tim_generator import get_geometry, get_itis_codes
+from flask import request, Flask
+from datetime import datetime
+
+app = Flask(__name__)
+
+log_level = os.environ.get('LOGGING_LEVEL', 'INFO')
+logging.basicConfig(format='%(levelname)s:%(message)s', level=log_level)
+
+def calculate_direction(coordinates):
+    try:
+        long_dif = coordinates[-1][0] - coordinates[0][0]
+        lat_dif = coordinates[-1][1] - coordinates[0][1]
+    except ValueError as e:
+        return "unknown"
+    except IndexError as e:
+        return "unknown"
+
+    if abs(long_dif) > abs(lat_dif):
+        if long_dif > 0:
+            # eastbound
+            direction = "I"
+        else:
+            # westbound
+            direction = "D"
+    elif lat_dif > 0:
+        # northbound
+        direction = "I"
+    else:
+        # southbound
+        direction = "D"
+    return direction
+
+def translate(rc_geojson):

```

Comment:
(blocking) Are there any unit tests for this method at the moment? It would be nice to be able to view an example of the expected input & output of this method.

Reply:
There were not - I've moved this method to tim_translator.py and added unit tests for this new file.